docs/topics/api/signing.rst

```diff
@@ -98,9 +99,10 @@ property.
                     }
                 ],
                 "passed_review": true,
+                "pk": "f68abbb3b1624c098fe979a409fe3ce9",
                 "processed": true,
                 "reviewed": true,
-                "url": "https://addons.mozilla.org/api/v3/addons/%40new-unlisted-api/versions/1.0/",
+                "url": "https://addons.mozilla.org/api/v3/addons/%40new-unlisted-api/versions/1.0/uploads/f68abbb3b1624c098fe979a409fe3ce9/",
```

Comment:
just a minor detail (and I know it's from another commit) but `@new-unlisted-api` is confusing as an example because it looks like a semantic URL qualifier saying "this add-on is unlisted." I'd suggest `@example-guid` or something more neutral.